src/app/app.module.ts

```diff
@@ -135,8 +135,7 @@ import { DashboardComponent } from './admin/dashboard/dashboard.component';
 import { AddTemplateComponent } from './admin/add-template/add-template.component';
 import { TestAndLaunchComponent } from './admin/test-and-launch/test-and-launch.component';
 import { TestAndVerifyComponent } from './admin/test-and-verify/test-and-verify.component';
-import { EditTemplateComponent } from './admin/edit-template/edit-template.component';
-import { DloginComponent } from './admin/dlogin/dlogin.component';
+import { ConfigurationsComponent } from './admin/configurations/configurations.component';
```

Comment:
@bhanuprakash-egov you are removing EditTemplateComponent and DloginComponent component from here. check once